docs/module_guides/evaluating/evaluating_evaluators_with_llamadatasets.md

```diff
@@ -0,0 +1,58 @@
+# Evaluating Evaluators with `LabelledEvaluatorDataset`'s
+
+The purpose of the llama-datasets is to provide builders the means to quickly benchmark
+LLM systems or tasks. In that spirit, the `LabelledEvaluatorDataset` exists to
+facilitate the evaluation of evaluators in a seamless and effortless manner.
+
+This dataset consists of examples that carries mainly the following attributes:
+`query`, `answer`, `ground_truth_answer`, and `reference_answer` along with some
```

Comment:
If we are evaluating evaluators, I would expect to see `evaluation_response` maybe? Or am I misunderstanding how it works?

Reply:
Yes, you're right. That's my bad, it shouldn't be `reference_answer` here but instead:
- `reference_score` and `reference_feedback` the expected outputs of an evaluator. So the reference here indicates the outputs of the reference evaluator.